build/build-bundle.js

```diff
@@ -92,6 +98,13 @@ async function browserifyFile(entryPath, distPath) {
     bundle = bundle.require(gatherer, {expose: gatherer.replace(driverPath, '../gather/')});
   });
 
+  // HACK: manually include the lighthouse-plugin-publisher-ads audits.
+  if (isDevtools(entryPath)) {
+    pubAdsAudits.forEach(pubAdAudit => {
+      bundle = bundle.require(pubAdAudit, {expose: '../../node_modules/' + pubAdAudit});
```

Comment:
add a note that all the audits are already prefixed with the module/plugin name?